src/RTTComms.cpp

```diff
@@ -111,28 +112,49 @@ namespace BrainCloud
             delete _socket;
             _socket = NULL;
         }
+
+        _rttConnectionStatus = RTTConnectionStatus::Disconnected;
```

Comment:
hummmm no. look at resetCommunication. you are in disconnecting at this point, we still have to clear the events.

Reply:
rightttt my mistake